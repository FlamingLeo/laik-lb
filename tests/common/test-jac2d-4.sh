#!/bin/sh
${LAUNCHER-./launcher} -n 4 ../../examples/jac2d -s 100 > test-jac2d-4.out
cmp test-jac2d-4.out "$(dirname -- "${0}")/test-jac2d-4.expected"
