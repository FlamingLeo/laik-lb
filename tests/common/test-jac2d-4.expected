100 x 100 cells (mem 0.2 MB), running 50 iterations with 4 tasks
Residuum after  1 iters: 30659.625000
Residuum after 11 iters: 59.794156
Residuum after 21 iters: 13.838317
Residuum after 31 iters: 7.521714
Residuum after 41 iters: 4.866531
Global value sum after 50 iterations: 24810.984574
